va_explorer/templates/va_data_management/show.html

```diff
@@ -31,9 +31,31 @@
 <div class="tab-content">
   <div class="tab-pane fade show active" id="record" role="tabpanel" aria-labelledby="record-tab">
     <div class="row mt-3">
-      {% if perms.va_data_management.change_verbalautopsy %}
-        <a class="btn btn-primary" href="{% url 'data_management:edit' id=id %}">Edit Record</a>
-      {% endif %}
+        <div class="va-options-container">
+            <div class="checkbox-container" align='center'>
+              <input type="checkbox" id="hidden-missing" name="hidden-missing" value="checked" checked="TRUE" style="margin-left: 20px;">
```

Comment:
Since both inputs have the same inline style, suggest extracting out into a class (like you did for va-options-container)

`style="margin-left: 20px;"`